debug: true
map_frame: map
odom_frame: odom
base_frame: base_footprint

robot_radius: 0.275

# sample
dock:
  dock_speed: 0.1
  dock_distance: 0.65
  pose_x: 3.634
  pose_y: -0.462
  pose_z: 0.000
  orientation_x: -0.001
  orientation_y: 0.002
  orientation_z: 0.316
  orientation_w: 0.949
